Azure/azure-cli
# [AppService] az webapp deployment github-actions add/remove: new commands

**Description**
New commands for adding/removing github actions to a webapp
- `az webapp deployment github-actions add`
- `az webapp deployment github-actions remove`

**Testing Guide**
Example usage:

- Add:

![image](https://user-images.githubusercontent.com/14339314/119858624-7f659600-bec9-11eb-92ab-a48f927932af.png)

Navigate to https://github.com/login/device to enter code
![image](https://user-images.githubusercontent.com/14339314/119858808-a91ebd00-bec9-11eb-9508-e88166c56087.png)

Authorize after entering in code
![image](https://user-images.githubusercontent.com/14339314/120008207-92db3480-bf8f-11eb-8905-bc41ca5cac0c.png)

- Remove:
![image](https://user-images.githubusercontent.com/14339314/119858991-d0758a00-bec9-11eb-90a8-039f47dc2328.png)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
